### DISCUSSION TITLE
Grafana Alerting - mTLS on webhook

### DISCUSSION THREAD

**Discussion by caveman280** [grafana/grafana#41035] [Alerting]
Hi there 👋 

We'd love to have mTLS support on the webhook from Grafana's unified alerting.

We'd like to POST directly to [Alerta](https://github.com/alerta/alerta), however, our alerta stack requires client certs for authentication.

Having mTLS would be great 🙂 

Cheers,
Thomas

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
